chrome/content_script.js

```diff
@@ -491,10 +487,15 @@ passwordalert.completePageInitializationIfReady_ = function() {
         securityEmailAddress: passwordalert.security_email_address_
       });
     }
-    chrome.runtime.sendMessage({action: 'savePossiblePassword'});
+    chrome.runtime.sendMessage({action: 'savePossiblePassword'}, response => {
+      if(chrome.runtime.lastError){
+        console.log('completePageInitializationIfReady_ ', chrome.runtime.lastError);
+      }
+    });
   }
 
   chrome.runtime.sendMessage({action: 'statusRequest'}, function(response) {
+    console.log("received response from service_worker: ", response);
```

Comment:
does this message contain any information that shouldn't be logged to the console?